.github/workflows/continous-integration.yml

```diff
@@ -1,24 +1,23 @@
 name: Run Tests
 
-on: 
+on:
   pull_request:
   push:
     branches: master
 jobs:
-  build:
-
+  test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8]
```

Comment:
Will this run all the tests (`flake8`, `black`, `mypy`) on all three Python versions? Or is this just ensuring that all three Python versions are available to `tox`?

Reply:
It runs all the tests on all three versions, `tox -e py,flake8,black,mypy`.

We could run just `tox -e py` on each version to just run the tests against each python version, and then run flake8, black and mypy on just one python version if we wanted.